test/sintel-captions.js

```diff
@@ -1,3 +1,4 @@
+// Seems to be too big to work with browserify so we stick it in window
```

Comment:
It may be possible to still use this in browserify by telling it to not parse this file. The option is called `noparse` in latest browserify: https://github.com/substack/node-browserify#browserifyfiles--opts